src/vm/docs/mod.rs

```diff
@@ -359,7 +359,7 @@ associated with the expressions' paired key name.",
 
 const TUPLE_GET_API: SpecialAPI = SpecialAPI {
     name: "get",
-    input_type: "KeyName, Tuple|Optional<Tuple>",
+    input_type: "KeyName and Tuple, or an optional Tuple",
```

Comment:
we should probably call "an optional Tuple" an "Optional(Tuple)", since Optionals are themselves a type.

Reply:
Ok, I hadn't realized that.